content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,225 @@ The tables below show changes made to the installation options used to customize
-| `pilot.env.GODEBUG` | `gctrace=1` |  |
-| `pilot.cpu.targetAverageUtilization` | `80` |  |
-| `pilot.nodeSelector` | `{}` |  |
-| `pilot.keepaliveMaxServerConnectionAge` | `30m` | `The following is used to limit how long a sidecar can be connectedto a pilot. It balances out load across pilot instances at the cost ofincreasing system churn.` |
+| `pilot.tolerations` | `[]` |  |
+| `pilot.podAntiAffinityLabelSelector` | `[]` |  |
+| `pilot.podAntiAffinityTermLabelSelector` | `[]` |  |
 
 ## Removed configuration options
 
-### Removed `ingress` key/value pairs
-
-| Key | Default Value | Description |
-| --- | --- | --- |
-| `ingress.service.ports.nodePort` | `32000` |  |
-| `ingress.service.selector.istio` | `ingress` |  |
-| `ingress.autoscaleMin` | `1` |  |
-| `ingress.service.loadBalancerIP` | `""` |  |
-| `ingress.enabled` | `false` |  |
-| `ingress.service.annotations` | `{}` |  |
-| `ingress.service.ports.name` | `http` |  |
-| `ingress.service.ports.name` | `https` |  |
-| `ingress.autoscaleMax` | `5` |  |
-| `ingress.replicaCount` | `1` |  |
-| `ingress.service.type` | `LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be` |  |
-
 ### Removed `servicegraph` key/value pairs
 
 | Key | Default Value | Description |
 | --- | --- | --- |
-| `servicegraph` | `servicegraph.local` |  |
-| `servicegraph.ingress` | `servicegraph.local` |  |
-| `servicegraph.service.internalPort` | `8088` |  |
-
-### Removed `telemetry-gateway` key/value pairs
-
-| Key | Default Value | Description |
-| --- | --- | --- |
-| `telemetry-gateway.prometheusEnabled` | `false` |  |
-| `telemetry-gateway.gatewayName` | `ingressgateway` |  |
-| `telemetry-gateway.grafanaEnabled` | `false` |  |
+| `servicegraph.ingress.enabled` | `false` |  |
+| `servicegraph.service.name` | `http` |  |
+| `servicegraph.replicaCount` | `1` |  |
+| `servicegraph.service.type` | `ClusterIP` |  |
+| `servicegraph.service.annotations` | `{}` |  |
+| `servicegraph.enabled` | `false` |  |
+| `servicegraph.image` | `servicegraph` |  |
+| `servicegraph.service.externalPort` | `8088` |  |
+| `servicegraph.ingress.hosts` | `servicegraph.local` | `Used to create an Ingress record.`  |
+| `servicegraph.nodeSelector` | `{}` |  |
+| `servicegraph.prometheusAddr` | `http://prometheus:9090` |  |
 
 ### Removed `global` key/value pairs
 
 | Key | Default Value | Description |
 | --- | --- | --- |
-| `global.hyperkube.tag` | `v1.7.6_coreos.0` |  |
-| `global.k8sIngressHttps` | `false` |  |
-| `global.crds` | `true` |  |
-| `global.hyperkube.hub` | `quay.io/coreos` |  |
-| `global.meshExpansion` | `false` |  |
-| `global.k8sIngressSelector` | `ingress` |  |
-| `global.meshExpansionILB` | `false` |  |
+| `global.proxy.tracer` | `"zipkin"` | `Specify which tracer to use. One of: lightstep, zipkin`  |
+| `global.controlPlaneSecurityEnabled` | `false` | `controlPlaneMtls enabled. Will result in delays starting the pods while secrets are propagated, not recommended for tests.`  |
```

Comment:
This also still exists... hope it'd be fixed with the tracer.

Reply:
Should be fixed now